gradio/components.py

```diff
@@ -802,9 +782,7 @@ def get_interpretation_scores(
     def generate_sample(self) -> float:
         return self.maximum
 
-        # Output Functionalities
-
-    def postprocess(self, y: float | None) -> float:
+    def postprocess(self, y: float | None) -> float | None:
```

Comment:
`postprocess()` always produces a float
```suggestion
    def postprocess(self, y: float | None) -> float:
```